openedx/adg/lms/webinars/views.py

```diff
@@ -16,8 +16,11 @@
 
 
 def webinar_description_page_view(request, pk):
-    # TODO This is a temp view and will be properly functional in LP-2623
-    return render(request, 'adg/lms/webinar/description_page.html', context={'pk': pk})
+    """
+       A view to get description about a specific webinar.
+    """
+
```

Comment:
No need for this blank line as we have used function based view.

Reply:
right (Y)